README.md

```diff
@@ -25,3 +25,57 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+
+# Runtime Analysis Answer:
+
+The time complexity of my divideAndConquerSum() is $\Theta(n)$.
+At the beginning of my algorithm, I check to see if the length of the
+array is 0 or 1. If the array is either of these lengths, then the execution
+time will be constant in the best case scenario. However, in the scenerio
+where the length is greater than 1, the complexity increases.
+The algorithm will always divide the array up into thirds and process
+its own portion of the array until it reaches the base cases where the 
+sub-array size is 0 or 1, in which it is added with two addition calls (hence 
+why $\Theta$ contains the value 2 rather than the value of 1).
+
+Recurrence Relation for divideAndConquerSum():\
+$T(n) = 3T(n/3) + \Theta(2)$
+
+###### Solving for the time complexity:
+- First we expand the recurrence relation:\
+    $T(n) = 3(3T(n/9) + \Theta(2)) + \Theta(2)$\
+    $T(n) = 9T(n/9) + 3\Theta(2) + \Theta(2)$\
+    $T(n) = 18T(n/18) + 9\Theta(2) + 3\Theta(2) + \Theta(2)$
+
+- Then we can generate a pattern:\
+    $T(n) = 3^k * T(n/3^k) + k * \Theta(2)$
+
+- In order to get T(1), we can plug in $k = log{_3}n$:\
+    $T(n) = n * T(1) + log{_3}n * \Theta(2)$
+
+- We know that the base case for T(1) is a constant time operation, so we 
+    can state that $T(1) = \Theta(1)$:\
+    $T(n) = n * \Theta(1) + log{_3}n * \Theta(2)$\
+    Simplified:\
+    $T(n) = \Theta(n) + \Theta(2log{_3}n)$
+
+- In this asymptotic analysis, $\Theta(2log{_3}n)$ is negligible, therefore
+    the final time complexity is $T(n) = \Theta(n)$
```

Comment:
Incorrect notation here.

Reply:
Could you be more specific? I don't understand what the error is.

Reply:
What symbol did we use in class to denote membership in a complexity class?